src/components/drag-and-drop/custom-drag-layer/custom-drag-layer.js

```diff
@@ -82,7 +86,7 @@ class CustomDragLayer extends React.Component {
 
   render() {
     return (
-      <div className='custom-drag-layer'>
+      <div className={ this.getKlassName(this.props) }>
```

Comment:
as you have set this up to pass an argument to the function, it might as well send the relevant arg: `this.getKlassName(this.props.className)`

Reply:
alternatively your function has access to `this` - but I guess that is not as functional!

Reply:
I styled access to props to be similar to `this.getItemStyles(this.props)` in this file.